BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudCustomEntity.cs

```diff
@@ -0,0 +1,300 @@
+        /// </remarks>
+        /// <param name="entityType">
+        /// </param>
+        /// <param name="entityId">
+        /// </param>
+        /// <param name="version">
+        /// </param>
+        /// <param name="dataJson">
+        /// </param>
+        /// <param name="acl">
+        /// </param>
+        /// <param name="timeToLive">
+        /// </param>
+        /// <param name="success">
+        /// The success callback.
+        /// </param>
+        /// <param name="failure">
+        /// The failure callback.
+        /// </param>
+        /// <param name="cbObject">
+        /// The user object sent to the callback.
+        /// </param>
+        public void UpdateCustomEntity(
+        string entityType,
+        string entityId,
+        int version,
+        string dataJson,
+        string acl,
+        string timeToLive,
+        SuccessCallback success = null,
+        FailureCallback failure = null,
+        object cbObject = null)
+        {
+            Dictionary<string, object> data = new Dictionary<string, object>();
+            data[OperationParam.CustomEntityServiceEntityType.Value] = entityType;
+            data[OperationParam.CustomEntityServiceEntityId.Value] = entityId;
+            data[OperationParam.CustomEntityServiceVersion.Value] = version;
+            data[OperationParam.CustomEntityServiceDataJson.Value] = JsonReader.Deserialize<Dictionary<string, object>>(dataJson);
+            data[OperationParam.CustomEntityServiceAcl.Value] = JsonReader.Deserialize<Dictionary<string, object>>(acl);
+            data[OperationParam.CustomEntityServiceVersion.Value] = timeToLive;
+
+            ServerCallback callback = BrainCloudClient.CreateServerCallback(success, failure, cbObject);
+            ServerCall sc = new ServerCall(ServiceName.CustomEntity, ServiceOperation.UpdateCustomEntity, data, callback);
+            _client.SendRequest(sc);
+        }
+
+        
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <remarks>
+        /// Service Name - CustomEntity
+        /// Service Operation - UpdateCustomEntityFields
+        /// </remarks>
+        /// <param name="context">
+        /// </param>
+        /// <param name="pageOffset">
+        /// </param>
+        /// <param name="success">
+        /// The success callback.
+        /// </param>
+        /// <param name="failure">
+        /// The failure callback.
+        /// </param>
+        /// <param name="cbObject">
+        /// The user object sent to the callback.
+        /// </param>
+        public void UpdateCustomEntityFields(
+        string entityType,
+        string entityId,
+        int version,
+        string fieldsJson,
+        SuccessCallback success = null,
+        FailureCallback failure = null,
+        object cbObject = null)
+        {
+             Dictionary<string, object> fields = JsonReader.Deserialize<Dictionary<string, object>>(fieldsJson);
```

Comment:
can we be consistent here? All the other calls with Json are the one line

data[OperationParam.CustomEntityServiceFieldsJson.Value] = JsonReader.Deserialize<Dictionary<string, object>>(fieldsJson);